client/src/components/JurisdictionAdmin/BatchInventory.tsx

```diff
@@ -1,11 +1,245 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { H2, H3, Button, H4, Colors, Icon } from '@blueprintjs/core'
+import { QueryObserverResult } from 'react-query'
+import styled from 'styled-components'
 import { Wrapper, Inner } from '../Atoms/Wrapper'
+import FileUpload from '../Atoms/FileUpload'
+import { ApiError } from '../../utils/api'
+import { IFileInfo } from '../useCSV'
+
+const STEPS = [
+  'Upload Election Results',
+  'Inventory Batches',
+  'Download Audit Files',
+] as const
+
+const StepContainer = styled.div`
+  background-color: ${Colors.LIGHT_GRAY5};
+`
+
+const StepPanel = styled.div`
+  padding: 50px 20px;
+  min-height: 250px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+`
+
+const StepActions: React.FC<{
+  left?: React.ReactElement
+  right?: React.ReactElement
+}> = ({ left, right }) => (
+  <div
+    style={{
+      display: 'flex',
+      padding: '20px',
+      backgroundColor: Colors.LIGHT_GRAY2,
+      justifyContent: 'space-between',
+    }}
+  >
+    {left ?? <div />}
+    {right ?? <div />}
+  </div>
+)
+
+const UploadElectionResultsStep: React.FC<{ nextStep: () => void }> = ({
+  nextStep,
+}) => {
+  return (
+    <StepContainer>
+      <StepPanel>
```

Comment:
Love the wizard concept! I like that the components are composable and leave state management to the consumer

A problem with the first iteration of the wizard component at Clever was the fact that it tried to bake state management into the component. The component interface grew really complex over time as it tried to support an ever growing number of state management cases. The second iteration of that component looked more like what you've begun hashing out in this PR

Reply:
That's a great lesson!

One pattern I've been experimenting with is using hooks for state management and stateless components for UI (e.g. the `useConfirm` hook and `Confirm` component). I was inspired by react-hook-form, which only does form state management and plugs into whatever UI you want. So we could potentially also try building a `useSteps` hook to manage state if we wanted.

Reply:
Ooh that's an interesting possibility!